src/index.js

```diff
@@ -100,6 +124,11 @@ function activateCheckboxListeners() {
   });
 }
```

Comment:
The functions activateMoreListeners, activateCheckboxListeners, activateTaskInputListeners, and activateCompletedListener share similar patterns. Consider creating a generic function to reduce duplication.